examples/default.yaml

```diff
@@ -103,6 +103,11 @@ mounts:
 # 🟢 Builtin default: "reverse-sshfs" (for QEMU), "virtiofs" (for vz)
 mountType: null
 
+# Enable inotify support for mounted directories (EXPERIMENTAL)
+# 🟢 Builtin default: Disabled by default
+mountInotify: null
```

Comment:
Wondering if this should be configurable per `[]mounts` entry

Reply:
I think that would be good, but could also be added later.

Ideally we should have the global option, and then a way to override it at the individual mount level.

Reply:
>  but could also be added later

Exactly my thought as well. Based on need we could add it later as well